sql/2019/08_Security/08_23-24.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
```

Comment:
What about a metric testing for how many sites meet the requirements of being in chrome's preload list? Could be interesting to see how many are ready to be added vs how many are in the preload list.

https://hstspreload.org/

Reply:
Good idea!